console/apps/webapp/public/config.js

```
@@ -1,15 +1,40 @@
+    // Disable strict ID token validation for providers with non-standard issuers 
+    // (e.g., Thunder uses "thunder" instead of a URL)
+    // Set VALIDATE_ID_TOKEN=true for providers that comply with OIDC standards (e.g., Asgardeo)
+    validateIDToken: '' === 'true',
+    // Clock tolerance (in seconds) to handle time skew between client and server
+    // Prevents token validation failures due to minor time differences
+    clockTolerance: 300
   },
-  disableAuth: true,
-  apiBaseUrl: "http://localhost:9000",
-  instrumentationUrl: "$INSTRUMENTATION_URL",
+  disableAuth: 'true' === 'true',
```

Biome (2.4.4)

[error] Comparing to itself is potentially pointless.

(lint/suspicious/noSelfCompare)